docker/maistra-builder_2.2.Dockerfile

```diff
@@ -70,7 +70,7 @@ RUN curl -sfL https://download.docker.com/linux/centos/docker-ce.repo -o /etc/yu
                    autoconf automake libtool cmake python2 python3 libstdc++-static \
                    gcc-toolset-9 gcc-toolset-9-libatomic-devel gcc-toolset-9-annobin gcc-toolset-9-libasan-devel \
                    java-11-openjdk-devel jq file diffutils lbzip2 annobin-annocheck \
-                   clang llvm lld ruby-devel zlib-devel openssl-devel python2-setuptools \
+                   clang llvm lld compiler-rt ruby-devel zlib-devel openssl-devel python2-setuptools \
```

Comment:
Would it be worth it to somehow visually separate this addition from the rest .. or even create a separate dockerfile layered on top of this one as a starting point for a dev image? I think this addition is non-essential for "regular/optimized" builds.. having a clear split between essential and non-essential (when reasoning about the regular build flow) might be useful? Feel free to push back, just thinking out loud.